src/edie/vocabulary.py

```diff
@@ -0,0 +1,6 @@
+SIZE_OF_DICTIONARY = "sizeOfDictionary"
+AVG_DICTIONARY_SIZE = 'avg_dictionary_size'
+
+# REPORTING PROPERTIES
+AGGREGATION_METRICS = 'aggregation_metrics'
+DICTIONARY_SIZE = 'dictionary_size'
```

Comment:
The combination of snake_case and CamelCase here is a bit ugly, do we want to fix this?

Reply:
Yes,  i agree. 
It's being mixed all over the place at the moment, and the naming should be consistent.
I will make a separate issue for this.